src/components/ImgLoading/styles.less

```diff
@@ -0,0 +1,41 @@
+@keyframes pulse {
+  0% {
+    transform: rotate(0);
+  }
+  50% {
+    transform: rotate(-180deg);
+  }
+  100% {
+    transform: rotate(-360deg);
+  }
+}
+.loading-image-default {
+  width: 72px;
+  height: 72px;
+  animation: pulse 800ms linear infinite;
+}
+
+.img-loading-wrapper {
+  background: #aedffe80;
```

Comment:
here

Reply:
It will be improved in the next version.